src/validation/__tests__/validation.js

```diff
@@ -65,9 +65,9 @@ describe('Validate: Supports full validation', () => {
     );
 
     expect(errors).to.deep.equal([
-      { message: 'Cannot query field "catOrDog" on "QueryRoot".' },
-      { message: 'Cannot query field "furColor" on "Cat".' },
-      { message: 'Cannot query field "isHousetrained" on "Dog".' }
+      { message: 'Cannot query field "catOrDog" on type "QueryRoot".' },
+      { message: 'Cannot query field "furColor" on type "Cat".' },
+      { message: 'Cannot query field "isHousetrained" on type "Dog".' }
```

Comment:
Probably; if we do so, I definitely need to add a dedicated test for `undefinedField` (I'll do that anyway).